src/js/components/Drop/DropContainer.js

```diff
@@ -30,6 +30,49 @@ const preventLayerClose = (event) => {
   }
 };
 
+// Gets the closest ancestor positioned element
+const getParentNode = (element) =>
+  element.offsetParent ?? element.parentNode;
+
+// return the containing block
+const getContainingBlock = (element) => {
+  const isIE = window.navigator.userAgent.toLowerCase().includes('trident');
+  if (isIE && element instanceof window.HTMLElement) {
+    // In IE 9, 10 and 11 fixed elements containing block is always established
+    // by the viewport
+    const css = window.getComputedStyle(element);
+    if (css.position === 'fixed') {
+      return null;
+    }
+  }
```

Comment:
Not sure if we need to include this since internet explorer's end of life is June 15, 2022.

Reply:
you're right